src/clients/search/search.spec.ts

```diff
@@ -0,0 +1,5 @@
+describe('search client', () => {
```

Comment:
Once again, obviously incomplete but in the interest of time we should go with this.